app/code/core/Mage/Wishlist/Controller/Abstract.php

```diff
@@ -85,7 +85,10 @@ public function allcartAction()
 
         $qtysString = $this->getRequest()->getParam('qty');
         if (isset($qtysString)) {
-            $qtys = array_filter(json_decode($qtysString), '\strlen');
+            $qtys = array_filter(json_decode($qtysString), fn($tmpString) => strlen($tmpString ?? ''));
```

Comment:
I vote line88 with arrow function, which is the same as line89.

Reply:
I also vote for 88 and removing the 3 lines that are commented out. I'd use a shorter variable name instead of $tmpString but ok who cares